### DISCUSSION TITLE
Wrong version of dbt used by dbt power user

### DISCUSSION THREAD

**Discussion by winzee** [AltimateAI/vscode-dbt-power-user#584] [Q&A] [answered]
Hi folks,

I suddenly started getting this weird error in one of my projects. If I run anything through DBT Power User, I can see it runs DBT v1.0.0. If I run it through the CLI, I get 1.5.0 (which is what I installed in my venv). So using DBT Power User to run anything I get a message like:

```
> Executing task: dbt run --select stg_ab_testing_results
18:03:57  Running with dbt=1.0.0
18:03:58  Encountered an error:
Runtime Error
  Failed to read package: Runtime Error
    This version of dbt is not supported with the 'dbt_utils' package.
      Installed version of dbt: =1.0.0
      Required version of dbt for 'dbt_utils': ['>=1.3.0', '<2.0.0']
    Check the requirements for the 'dbt_utils' package, or run dbt again with --no-version-check
    
  
  Error encountered in /Users/lmg/code/app-analytics/dbt_packages/dbt_utils/dbt_project.yml
Error encountered in /Users/lmg/code/app-analytics/dbt_packages/dbt_utils
```

Where is DBT Power User getting that reference to dbt 1.0.0?  I run my dbt project in a venv environment, but even if I deactivate it in my CLI, I get DBT 1.5.0 by default so that is not the problem.  My other projects work just fine on dbt 1.3...

Most importantly: how can I change DBT Power User to use the dbt version in my venv?

Thanks!

**Comment by anandgupta42** [comment, marked as answer]
@winzee It seems the dbt power user is pointing to a different venv that has dbt 1.0.0. Please check the [guide](https://github.com/AltimateAI/vscode-dbt-power-user#select-the-python-interpreter-that-has-dbt-installed) for more details. 

If you want to create a new venv please use [this](https://www.loom.com/share/b1167b0d583c4dc49916939846e82ddb?sid=f1acf7da-5d51-4f19-9e6a-ea8d3821ac3c) video for further instruction.

**Reply by winzee** [reply]
Thank you, it pointed me in the right direction. Even if I deleted and recreated the venv, DBT power user kept using an old version of DBT. I never figured out why. It was not a DBT environment variable. There is also no DBT installed outside of a venv environment on my computer. In the end, I had to create a new empty project directory, pull from git and reinstall from there and it worked.
